src/k8s/requests/v2v/startV2VvmwareController.js

```diff
@@ -1,32 +1,64 @@
-import { DeploymentModel, RoleModel, ServiceAccountModel, RoleBindingModel, findModel } from '../../../models';
+import {
+  DeploymentModel,
+  RoleModel,
+  ServiceAccountModel,
+  RoleBindingModel,
+  findModel,
+  ConfigMapModel,
+} from '../../../models';
 import { buildServiceAccount, buildServiceAccountRoleBinding } from '../../objects';
 import { buildVmWareRole, buildVmWareDeployment } from '../../objects/v2v/vmware';
-import { V2VVMWARE_DEPLOYMENT_NAME } from './constants';
+import {
+  V2VVMWARE_DEPLOYMENT_NAME,
+  VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAME,
+  VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAMESPACE,
+} from './constants';
 import { getDeploymentContainer } from '../../../selectors/deployment';
 import { getContainerImage } from '../../../selectors/pod';
 import { buildAddOwnerReferencesPatch, buildOwnerReference } from '../../util';
 import { getName } from '../../../selectors';
-import { getKubevirtV2vVmwareContainerImage } from '../../../config';
+import { getKubevirtV2vVmwareContainerImage, getV2vImagePullPolicy } from '../../../selectors/v2v';
 
 const { info } = console;
 
 const OLD_VERSION = 'OLD_VERSION';
 
+// prevent parallel execution of startV2VVMWareController()
+const semaphors = {};
+
 // The controller is namespace-scoped, especially due to security reasons
 // Let's make sure its started within the desired namespace (which is not by default).
 // The V2VVmware CRD is expected to be already created within the cluster (by Web UI installation)
+// TODO: The controller should be deployed by a provider and not via following UI code.
 export const startV2VVMWareController = async ({ namespace }, { k8sGet, k8sCreate, k8sKill, k8sPatch }) => {
   if (!namespace) {
     throw new Error('V2V VMWare: namespace must be selected');
   }
 
+  if (semaphors[namespace]) {
+    // eslint-disable-next-line no-console
+    console.info(`startV2VVMWareController for "${namespace}" namespace already in progress. Skipping...`);
```

Comment:
not sure if this message is necessary

Reply:
This functionality caused a lot of confusion so far, so I prefer to keep it for visibility.
No matter it's not a simple task, the user is unaware of it. 
Might help us with debugging ....